pkg/controller/staticroute/staticroute_controller.go

```diff
@@ -157,16 +177,16 @@ func (r *ReconcileStaticRoute) Reconcile(request reconcile.Request) (reconcile.R
 	// If "gateway" is empty, we'll create the route through the default private network gateway
 	gateway := rw.getGateway()
 	if gateway == nil {
-		gwRoute, err := netlink.RouteGet(net.IP{10, 0, 0, 1})
+		defaultGateway, err := params.options.RouteGet()
 		if err != nil {
 			reqLogger.Error(err, "")
 			return reconcile.Result{}, err
 		}
-		reqLogger.Info(fmt.Sprintf("* %+v", gwRoute[0].Gw))
-		gateway = gwRoute[0].Gw
+		reqLogger.Info(fmt.Sprintf("* %+v", gateway))
```

Comment:
Don't we need to change the order of the line 185 and 186?

Reply:
fixed